.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js Workflow
+
+on:
+  push:
+  pull_request:
```

Comment:
unnecesary if you already have the push trigger

Reply:
you should delete the pull request trigger for the push trigger with the master branch excluded